src/components/Checkbox/Checkbox.tsx

```diff
@@ -102,7 +110,12 @@ const Input = styled.input`
   &[disabled] {
     pointer-events: none;
   }
+
+  &:focus + span {
+    box-shadow: 0 0 0 2px rgba(255, 0, 0, 0.85); /* FIXME focus style */
```

Comment:
Use the color here.
`rgba(0, 165, 171, 0.5)`

Reply:
Thank you for fix color!

If `rgba(0, 165, 171, 0.5)` mean half transparent `palette.Main`, use polished opacity method like `opacity(0.5, ${palette.Main}) is better than just `rgba()`. Additionally looks good that to define outline color to Palette Property as `Outline`. What do you think about ?

https://polished.js.org/docs/#opacify

Reply:
That's right ✨
Since `polished` is already installed, you should use it.
Thank you for the exact proposal!!

Reply:
@nabeliwo I add commit for translate to half transparent color from `Main` by transparentize (I made mistake `opacify` as make transparent method.) Please confirm again 🙏